packages/oui-numeric/src/numeric.html

```diff
@@ -0,0 +1,29 @@
+<label ng-attr-for="{{:: $ctrl.id }}"
```

Comment:
To be discussed on our next daily if we still need the prefix `data-`.
It matters only for the validation